src/dra/8D3E8.c

```diff
@@ -1319,5 +1398,3 @@ void func_80134E64(void) {
 
 void func_80134F50();
 INCLUDE_ASM("dra/nonmatchings/8D3E8", func_80134F50);
-
-const u32 rodataPadding_jpt_8012D4D8 = 0;
```

Comment:
I missed this originally, do you know why do we no longer need this padding?

Reply:
That's because of pulling in the rodata for this new function. Before we only had the rodata for func_8012D3E8, but now we also have func_8012EF2C's rodata getting parsed in.

The jump table for func_8012D3E8 ends with 4 bytes of zeros, while the jump table for func_8012EF2C does not. I don't know the **exact** details of why that is or anything, but that's the reason. Before, our last rodata ended with a set of zeros, and now, our last rodata does not. 
![image](https://github.com/Xeeynamo/sotn-decomp/assets/15314202/30054ef9-479a-49ab-88a4-bb6b441932ed)